python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -309,7 +313,8 @@ class LocalRunner(RPCRunner):
         Whether check correctness after measurement. This will use llvm cpu target to
         call your template and get the reference output.
         This can work for TOPI templates, but may not work for your custom template.
-
+    enable_cpu_cache_flush: bool
+        Whether to enable cpu cache flush
```

Comment:
Maybe explicitly mention that this has no effect on GPU tasks?

Reply:
Thanks, I will update the doc